app/views/layouts/_navbar.html.erb

```diff
@@ -0,0 +1,49 @@
+<nav>
+ <div class="container">
```

Comment:
You might want to use slim or haml for your views. It is nicer than erb :)

Reply:
We wanted to follow Facebook's example of having the signup/login page as the home page. This means we had to change a few default routes to match this specific case.